src/StatusQ/Controls/StatusBaseInput.qml

```diff
@@ -220,7 +220,12 @@ Item {
                             anchors.left: parent.left
                             anchors.right: parent.right
                             anchors.rightMargin: root.rightPadding
-                            anchors.verticalCenter: parent.verticalCenter
+                            anchors.top: parent.verticalAlignment == TextEdit.AlignTop?
+                                                        parent.top : undefined
+                            anchors.bottom: parent.verticalAlignment == TextEdit.AlignBottom?
+                                                        parent.bottom : undefined
+                            anchors.verticalCenter: parent.verticalAlignment == TextEdit.AlignVCenter?
+                                                        parent.verticalCenter : undefined
```

Comment:
I am wondering if this whole positioning could be simplified to:
```
anchors.fill: edit
verticalAlignment: Edit.AlignVCenter
```

Reply:
Looking into the code, should be doable, will try later...